doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -326,234 +345,308 @@ Per-worker relations expose the same data as their global counterparts, but have
 
 The `mz_dataflow_arrangement_sizes` view describes how many records and batches
 are contained in operators under each dataflow.
 
-| Field     | Type       | Meaning                                                                      |
-|-----------|------------|------------------------------------------------------------------------------|
-| `id`      | [`bigint`] | The ID of the [dataflow]. Corresponds to [`mz_dataflows.id`](#mz_dataflows). |
-| `name`    | [`bigint`] | The name of the object (e.g., index) maintained by the dataflow.             |
-| `records` | [`bigint`] | The number of records in all arrangements in the dataflow.                   |
-| `batches` | [`bigint`] | The number of batches in all arrangements in the dataflow.                   |
+<!-- RELATION_SPEC mz_internal.mz_dataflow_arrangement_sizes -->
+| Field     | Type        | Meaning                                                                      |
+|-----------|-------------|------------------------------------------------------------------------------|
+| `id`      | [`uint8`]   | The ID of the [dataflow]. Corresponds to [`mz_dataflows.id`](#mz_dataflows). |
+| `name`    | [`text`]    | The name of the object (e.g., index) maintained by the dataflow.             |
+| `records` | [`numeric`] | The number of records in all arrangements in the dataflow.                   |
+| `batches` | [`numeric`] | The number of batches in all arrangements in the dataflow.                   |
 
 ### `mz_message_counts`
 
 The `mz_message_counts` view describes the messages sent and received over the [dataflow] channels in the system.
 
-| Field              | Type         | Meaning                                                                                   |
-| ------------------ | ------------ | --------                                                                                  |
-| `channel_id`       | [`bigint`]   | The ID of the channel. Corresponds to [`mz_dataflow_channels.id`](#mz_dataflow_channels). |
-| `sent`             | [`bigint`]   | The number of messages sent.                                                              |
-| `received`         | [`bigint`]   | The number of messages received.                                                          |
+<!-- RELATION_SPEC mz_internal.mz_message_counts -->
+| Field              | Type        | Meaning                                                                                   |
+| ------------------ |-------------| --------                                                                                  |
+| `channel_id`       | [`uint8`]   | The ID of the channel. Corresponds to [`mz_dataflow_channels.id`](#mz_dataflow_channels). |
+| `sent`             | [`numeric`] | The number of messages sent.                                                              |
+| `received`         | [`numeric`] | The number of messages received.                                                          |
```

Comment:
I was wondering about the type of these as well and guess they also should not be `numeric`.

Reply:
Yup, I agree! Listed in #20522.